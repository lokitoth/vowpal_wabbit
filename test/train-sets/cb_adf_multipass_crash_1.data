shared | a:1 b:0.5
0:0.1:0.75 | a:0.5 b:1 c:2
| a:1 c:3

shared | s_1 s_2
0:1.0:0.5 | a:1 b:1 c:1
| a:0.5 b:2 c:1
