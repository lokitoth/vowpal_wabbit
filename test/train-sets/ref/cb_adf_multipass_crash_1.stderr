Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
decay_learning_rate = 1
using cache_file = train-sets\cb_adf_multipass_crash_1.data.cache
ignoring text input in favor of cache input
num sources = 1
Enabled reductions: gd, scorer, csoaa_ldf, cb_adf, shared_feature_merger
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
0.133333 0.133333            1            1.0 0:0.1:0.75        0:0...        7
